web/src/click_handlers.js

```diff
@@ -835,6 +836,21 @@ export function initialize() {
         dark_theme.disable();
     });
 
+    // PILLS
+    $("body").on("click", ".pill", function (e) {
+        const participant_user_id = Number.parseInt($(this).attr("data-user-id"), 10);
+        e.stopPropagation();
+
+        // if the pill is not a user pill and does not have a user-id
+        // we don't need to show the user card so we return early
+        if (Number.isNaN(participant_user_id)) {
+            return;
+        }
+
+        const user = people.get_by_user_id(participant_user_id);
+        popovers.show_user_info_popover(this, user);
```

Comment:
I think we should have this in `input_pill.js` where have the click handler for `x` button.

Reply:
Updated. Thank you @sahil839